tests/operators/test_great_expectations.py

```diff
@@ -122,16 +364,15 @@ def test_great_expectations_operator__validation_failure_raises_exc():
 def test_great_expectations_operator__validation_failure_logs_warning(caplog):
     operator = GreatExpectationsOperator(
         task_id="task_id",
-        expectation_suite_name="taxi.demo",
-        batch_kwargs={"path": bad_data_file, "datasource": "data__dir"},
         data_context_root_dir=ge_root_dir,
+        checkpoint_name="taxi.fail.chk",
         fail_task_on_validation_failure=False,
     )
     operator._log = logging.getLogger("my_test_logger")
     caplog.set_level(level="WARNING", logger="my_test_logger")
     caplog.clear()
     result = operator.execute(context={})
-    assert result["success"] == False
+    assert not result["success"]
```

Comment:
If we wanted to check for `False` (as opposed to possibly `None`) we could use `is False`? I don't think this matters unless there's a change that `result["success"]` contains a non-boolean value

Reply:
Interesting - I think this should always be boolean, but it would be safer to use `is False`. I'll update.